docs/end2end-TrecQA-eval.md

```diff
@@ -13,7 +13,7 @@ if you need help setting up the gateway.
 Run the following command to evaluate the end2end system:
 
 ```
-python3 src/main/python/run_trecqa.py
+python src/main/python/run_trecqa.py
```

Comment:
@snapbug we decided on using Python3 for all our systems. Therefore, I updated this line. It's part of the requirements list.